tests/Feature/BookTest.php

```diff
@@ -183,7 +183,41 @@ public function test_get_books_filtered_by_genre(): void
                                 ]);
                         });
                 });
-    }
+        }
+        
+        public function test_search_books(): void
+        {
+            $book = Book::factory()->create([
+                'title' => 'Harry Potter',
+                'author' => 'Harry Maguire',
+                'blurb' => 'Harry book',
+            ]);
+        
+            $response = $this->getJson('/api/books?search=Harry');
+        
+            $response
+                ->assertStatus(200)
+                ->assertJson(function (AssertableJson $json) {
+                    $json
+                        ->hasAll(['message', 'data'])
+                        ->has('data', 1, function (AssertableJson $json) {
+                            $json
+                                ->hasAll(['id', 'title', 'author','blurb', 'page_count', 'year', 'image', 'created_at', 'updated_at', 'genre_id', 'genre', 'reviews'])
+                                ->whereAllType([
+                                    'id' => 'integer',
+                                    'title' => 'string',
+                                    'author' => 'string',
+                                    'blurb' => 'string',
+                                    'claimed_by_name' =>'string',
+                                    'page_count' => 'integer',
+                                    'year' => 'integer',
+                                    'image' => 'string',
+                                    'genre_id' => 'integer',
+                                ]);
+                        });
+                });
+        }
```

Comment:
You should also add a couple of additional tests to make sure the search always works when used along side the claimed and genre filters.

I would suggest the following:

1.  Search with claimed filter
2.  Serach with genre filter
3.  Search with both claimed and genre filter